eisbuk-admin/firestore.rules

```diff
@@ -37,16 +37,17 @@ service cloud.firestore {
         );
     }
     match /organizations/{organization} {
+      allow create, update: if isAdmin(organization);
```

Comment:
Note to self: administrators have permission to remove themselves from an organization. Not a problem, just keeping it in mind. We don't even have any UI yet to manage admins.